tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=redefined-outer-name
+import tensorflow as tf
+import pytest
+
+
+@pytest.fixture(scope="session")
+def dataset():
+    mnist = tf.keras.datasets.mnist
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
+    x_train, x_test = x_train / 255.0, x_test / 255.0
+    yield (x_train, y_train), (x_test, y_test)
+
+
+@pytest.fixture(scope="session")
+def model():
+    model = tf.keras.models.Sequential(
+        [
+            tf.keras.layers.Flatten(input_shape=(28, 28)),
+            tf.keras.layers.Dense(128, activation="relu"),
+            tf.keras.layers.Dropout(0.2),
+            tf.keras.layers.Dense(10),
+        ]
+    )
+    loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
+    model.compile(
+        optimizer="adam",
+        loss=loss_fn,
+        metrics=["accuracy"],
+    )
+    yield model
```

Comment:
yield -> return?

Reply:
I took this from Prophet integration tests, but you are correct return is fine here.